documentation/release_notes.markdown

```diff
@@ -302,6 +302,73 @@ derive data from a PuppetDB export tarball.
 
 TODO
```

Comment:
@wkalt perhaps not, that's a placeholder (already) in master, though I think we may eventually want to move it back up near the top where we normally put the contributors?